public/views/projects-migrated.html

```diff
@@ -39,10 +33,22 @@ <h4 class="site-header">{{migratedProject.site_name}}</h4>
 				 <span ng-show="migratedProject.destination_url===null">Exported to <span ng-if="migratedProject.destination_type==='mailarchivezip'">zip</span><span ng-if="migratedProject.destination_type!=='mailarchivezip'">{{migratedProject.destination_type}}</span></span>
          <span ng-show="migratedProject.migrated_by"> by {{migratedProject.migrated_by}}</span>
 				</div>
-        <div class="col-md-6 col-lg-6 col-sm-6 col-xs-6" style="text-align:right">
-          <small ng-show="migratedProject.status.data.counts.failures !==0" class="export-fail">
-            There were {{migratedProject.status.data.counts.failures}} failures
+        <div class="col-md-6 col-lg-6 col-sm-6 col-xs-6 {{migratedProject.status.status}}" style="text-align:right">
+          <small ng-show="migratedProject.status.counts.successes ===0 && migratedProject.status.counts.errors ===0">
```

Comment:
Let say Mbox or google migration, if a email has error while formatting we just don't migrate the email to Google or don't put that in the Mbox file. We capture this in the Report in the `items []` . So error sending a single email don't stop the process just captured in the Report object